addons/account/tests/test_account_move_entry.py

```diff
@@ -637,6 +637,31 @@ def test_entry_reverse_storno(self):
             }
         ])
 
+    def test_entry_reverse_keeps_partner(self):
+        partner = self.env['res.partner'].create({
+            'name': 'Test Partner',
+        })
+        move = self.env['account.move'].create({
+            'move_type': 'entry',
+            'partner_id': partner.id,
```

Comment:
It doesn't really make sense to have a partner on a journal entry, generally.
The steps to reproduce talk about a bank statement, where it makes more sense (because it is a `_inherits`)
Please adapt the test to make it more logical regarding the business case